### PR TITLE
Fixes the tribal selection in the book and adds wayfarer armor back

### PR DESCRIPTION
**Pull request by OliOliOnsiPree**
## About The Pull Request
What this PR does is:
This PR fixes the tribal selection in the book instead of giving you half the recipes its gives you the full recipes for the tribe you selected, this also adds in wayfarer armor from the old ole dr2 ages, even though it is called western wayfarer gear, it still works plus there is lore behind it anywho. Most of the recipes are balanced around the other ones. Basically majority of this PR is fixes and the other half brings in two new sets of armor that the wayfarers used.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Western wayfarer gear into the recipes and turns it into wayfarer recipes under the book.
fix: All the tribal recipes should be given all the recipes of the tribe you selected if not then whats the point of the book.
/:cl:

### CODE REVIEW

File: icons/fallout/clothing/armored_heavy.dmi
Comment (Zero-Percent-Angel) [inline icons/fallout/clothing/armored_heavy.dmi:1]: 89% of the file removed in terms of file size seems a bit odd..

File: icons/fallout/clothing/armored_light.dmi
Comment (Zero-Percent-Angel) [inline icons/fallout/clothing/armored_light.dmi:1]: Ditto with this but for 90%.

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -1183,7 +1183,7 @@
 	granted_trait = null
 	pages_to_mastery = 0
 	time_per_page = 0
-
+/*
 /obj/item/book/granter/trait/tribaltraditions/attack_self(mob/user)
```
Comment (Zero-Percent-Angel) [inline code/game/objects/items/granters.dm:1187]: Seems odd just to be commenting out the attack self functionality of this, could you explain to me what you're trying to do? :)

**Comment by Zero-Percent-Angel** [comment]
Could you take some screen shots of them two DMI files for me?

**Comment by OliOliOnsiPree** [comment]
image
image

**Comment by LoadedGunius** [comment]
Love to see it.

**Comment by Zero-Percent-Angel** [comment]
Do they already have on mob icon states?

**Comment by OliOliOnsiPree** [comment]
> Do they already have on mob icon states?

Yes they do, they were already in the code from coyote.

**Pull request merged**